onos-topo/values.yaml

```diff
@@ -129,3 +226,5 @@ openidc:
 podSecurityContext:
   seccompProfile:
     type: "Unconfined"
+
+app_label: onos-topo
```

Comment:
I'm not sure why we have this label... this is information that can be retrieved from the chart info. We should remove it and remove the use of the `app` label.